lib/registry.js

```diff
@@ -60,7 +58,7 @@ function loadFromApi () {
   })
     .then(JSON.parse)
     .then(payload => payload.content)
-    .then(raw => new Buffer(raw, 'base64').toString())
+    .then(raw => Buffer.from(raw, 'base64').toString())
```

Comment:
Oh, just saw the `standard` error. `new Buffer()` was deprecated. Nevermind.

```
./lib/registry.js:63:18: 'new Buffer()' was deprecated since v6. Use 'Buffer.alloc()' or 'Buffer.from()' (use 'https://www.npmjs.com/package/safe-buffer' for '<4.5.0') instead. (node/no-deprecated-api)
```